### DISCUSSION TITLE
eza v0.10.9

### DISCUSSION THREAD

**Discussion by cafkafk** [eza-community/eza#175] [Announcements]
# Description

This was a relatively tame release, but there still were some interesting developments. One being the [matrix chat room](https://matrix.to/#/#eza:gitter.im) for coordinating development, on the initiative of @PThorpe92.

New stuff:
- The `--no-git` flag was added.

For contributors:
- We now have a [codeowners file](https://github.com/eza-community/eza/pull/164).

# Changelog

### Bug Fixes

- Respect git-repos flags

### Documentation

- Add badge for eza gitter/matrix room
- Fix matrix link
- Add ignored flags to readme
- Add ignored flags to manual
- Add ignored flags to help
- Add ignored flags to xtest

### Features

- `--no-git` option

### Miscellaneous Tasks

- Add funding.yml

### Build

- Add convco to dev

### Ci

- Create flakehub-publish-tagged.yml
- Add workflow_dispatch to flakehub-pub
- Edit workflow_dispath
- Refactor workflow_dispath
- Refactor workflow_dispath
- Remove broken dispatch
- Add flakehub-backfill
- Add codeowners
- Add gierens as .deb codeowner

### Deps

- Change users depedency to uzers

## New Contributors
* @Aehmlo made their first contribution in https://github.com/eza-community/eza/pull/168

**Full Changelog**: https://github.com/eza-community/eza/compare/v0.10.8...v0.10.9

<hr /><em>This discussion was created from the release <a href='https://github.com/eza-community/eza/releases/tag/v0.10.9'>eza v0.10.9</a>.</em>

**Comment by nachtjasmin** [comment]
Just wanted to say thank you for this fork and the work you're putting into it 🥰

**Reply by cafkafk** [reply]
Thanks, that means a lot :heart:

**Comment by gierens** [comment]
deb repo updated ;)
